style.css

```diff
@@ -1,30 +1,120 @@
-#book-collection {
-  margin: 20px;
+body {
+  font-family: Arial, sans-serif;
 }
 
-#book-collection .book {
-  border-bottom: 1px solid black;
+h1 {
+  margin: 0;
+  padding: 1em;
+  text-align: center;
+  background-color: #ff4081;
+  color: #fff;
+}
+
+.book-form {
   display: flex;
   flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  margin-bottom: 20px;
 }
 
-button {
-  width: 10%;
-  margin: 20px;
-  margin-left: 70px;
+.book-form label {
+  margin-right: 10px;
 }
 
-#book-collection .book button {
-  cursor: pointer;
-  width: fit-content;
+#add-button {
+  margin-left: 10px;
+}
+
+#book-list {
+  width: 90vw;
+  margin: auto;
+  margin-top: 20px;
+}
```

Comment:
-[OPTIONAL] Good job styling your app. it looks nice but there is a small issue , when the user adds new books, the `add new book` form gets down in page and then the user needs to go down in he page every time to add a new book which I think is not a pleasant user experience. 
kindly consider applying a scroll method in `book list` container and giving it a fixed height. that would solve the problem.